influxdb/30-influxdb-client-pod.yaml

```diff
@@ -0,0 +1,11 @@
+
+apiVersion: v1
```

Comment:
That is nice!

Reply:
We might need a command  and arguments in the yaml to pass to the container that executes an sleep forever so the Pod doesn't terminate right away. 

For instance: https://github.com/storageos/use-cases/blob/9f91bae2bb25c0278974cc465d7b4865ec1eed16/mysql/30-mysql-client-pod.yaml#L11

Reply:
added